README.md

```diff
@@ -79,4 +79,4 @@ If you think this project is worth supporting financially please make a contribu
 
 ## Things that are currently annoying me
 
-[![](https://codescene.io/projects/697/status.svg) Get more details at **codescene.io**.](https://codescene.io/projects/697/jobs/latest-successful/results)
\ No newline at end of file
+[![](https://codescene.io/projects/697/status.svg) Get more details at **codescene.io**.](https://codescene.io/projects/697/jobs/latest-successful/results)

```

Comment:
![](https://codescene.io/projects/697/status-badges/code-health) Get more details at [**codescene.io**](https://codescene.io/projects/697/jobs/latest-successful/results).

Taking into account this Blame results: [Blame README.md](https://github.com/ThreeMammals/Ocelot/blame/develop/README.md), I would say that markdown code of the badge is outdated.
Check the latest docs regarding Badges: [Keep Tabs on the State of your Code with Badges — CodeScene 1 Documentation](https://codescene.io/docs/integrations/publish-status-badges.html)
I've double checked, and, Yes, current markdown is invalid!

Valid markdown of the Health Badge :point_down: 
```markdown
[![Ocelot Code Health](https://codescene.io/projects/697/status-badges/code-health)](https://codescene.io/projects/697)
```
#### Example of my project badge
[![CodeScene Code Health](https://codescene.io/projects/38939/status-badges/code-health)](https://codescene.io/projects/38939)

Finally, you need to change line 82 with this code:
```markdown
[![Ocelot Code Health](https://codescene.io/projects/697/status-badges/code-health)](https://codescene.io/projects/697) Get more details at [**codescene.io**](https://codescene.io/projects/697/jobs/latest-successful/results)
```
So, it will look like:
> [![Ocelot Code Health](https://codescene.io/projects/697/status-badges/code-health)](https://codescene.io/projects/697) Get more details at [**codescene.io**](https://codescene.io/projects/697/jobs/latest-successful/results)

Note, we don't see the badge because project settings is wrong. We need to ask Tom to fix project settings. It seems the badge is disabled.

Reply:
As suggested, done. Please review @raman-m